src/python/pants/option/option_types.py

```diff
@@ -311,6 +350,10 @@ class ShellStrOption(StrOption):
 
     option_type: Any = custom_types.shell_str
 
+class WorkspacePathOption(StrOption):
+    """A workspace path option."""
+
+    option_type: Any = custom_types.workspace_path
```

Comment:
This is missing tests

Reply:
I don't think it's a big deal, all the type does is rejects `/`